io/zenoh-link-commons/src/unicast.rs

```diff
@@ -45,6 +45,7 @@ pub trait LinkUnicastTrait: Send + Sync {
     fn get_dst(&self) -> &Locator;
     fn is_reliable(&self) -> bool;
     fn is_streamed(&self) -> bool;
+    fn get_interfaces(&self) -> Vec<String>;
```

Comment:
I wonder wether a more appropriate name could be `get_interfaces_names` since what we return here is the symbolic name of a network interface.
Interfaces could also return their indexes as presented by the OS, so I believe it's good to disambiguate. Opinions?

Reply:
Fine, but will be better `get_interface_names`